includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -131,6 +132,8 @@ function get_sortable_columns() {
 			case 'courses':
 				$columns = array(
 					'title'           => array( 'title', false ),
+					'students'        => array( 'students', false ),
```

Comment:
I don't quite understand why `students` and `lessons` are being added as sortable columns when they aren't part of the courses table? What I was suggesting [here](https://github.com/Automattic/sensei/pull/4917#discussion_r831502729) is to remove those columns altogether from this function so that they don't render as links in the column header.

Reply:
I've messed this up when reverting stuff I've added there. I removed everything now. Completions were also not working: [commit](https://github.com/Automattic/sensei/pull/4917/commits/34a53e73bb95283d7a91a16e047dee3411cd0b5a)